app.js

```diff
@@ -61,12 +61,19 @@ app.post('/api/newuser', index.newUser); // new user details needing to be added
 app.get('/api/getUser', index.getUser); //see who's logged in
 app.get('/api/getSurvey', index.getSurvey); //get a survey the user hasn't taken
 
-app.post('/login',
-  passport.authenticate('local', { failureRedirect: '/' }),
-  function (req, res) {
-    res.redirect('/');
-  }
-);
+app.post('/login', function(req, res, next) {
+  passport.authenticate('local', function(err, user, info) {
+    if (err) { return next(err); }
+    if (!user) {
+      console.log('no user');
+      return res.json({ loggedIn:false });
+    }
+    req.logIn(user, function(err) {
+      if (err) { return next(err); }
+      return res.json({ loggedIn:true });
+    });
+  })(req, res, next);
+});
```

Comment:
This way allows us to handle login failure ourselves instead of just setting a "failureRedirect" URL